components/builder-api/habitat/default.toml

```diff
@@ -1,3 +1,6 @@
+[srv]
+port = 9636
```

Comment:
This line was the exact one I referenced above. I would make it not `srv`, since we have a namespace for that already.